### PR TITLE
[REF] simple extraction of getFieldAlterSQL

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Simple extraction of getFieldAlterSQL

Before
----------------------------------------
No way to get field alter sql without table alter 'header' prepended

After
----------------------------------------
Can retrieve field specific

Technical Details
----------------------------------------
Simple extraction

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Core/BAO/SchemaHandler.php [hunk 1]
```diff
@@ -758,7 +758,23 @@ public static function createMissingIndices($missingIndices) {
    */
   public static function buildFieldChangeSql($params, $indexExist) {
     $sql = str_repeat(' ', 8);
-    $sql .= "ALTER TABLE {$params['table_name']}";
+    $sql .= "ALTER TABLE {$params['table_name']} ";
+    return $sql . self::getFieldAlterSQL($params, $indexExist, $sql);
```
Comment (pradpnayak) [inline CRM/Core/BAO/SchemaHandler.php]: third parameter in getFieldAlterSQL() needed? Can't see any 3rd parameter defined in the function

Comment (eileenmcnaughton) [reply CRM/Core/BAO/SchemaHandler.php]: indeed I later removed it but not in this PR - fixed

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Pull request merged**